repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/examples/container.cfg.xml

```diff
@@ -32,4 +32,10 @@
 -->
         </ssl-configuration>
     </deployment-config>
+
+    <cluster-config cluster-id="internal-cluster">
```

Comment:
Oh, that would also require this service listening listening for changes to the system model.